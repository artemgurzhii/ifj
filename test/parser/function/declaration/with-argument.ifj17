function foo(bar as string) as string
end
