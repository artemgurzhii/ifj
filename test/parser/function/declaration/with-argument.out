(function foo -> (id string)
  (decl
    (id bar)
    : (id string))
)
